test/controllers/searches_controller_test.rb

```diff
@@ -0,0 +1,20 @@
+require 'test_helper'
+
+class SearchesControllerTest < ActionController::TestCase
+  def search
+    @_search ||= searches(:ranged)
+  end
+
+  test 'should get show' do
+    get :show, id: search.id
+    assert_response :success
+    assert assigns(:search)
+  end
+
+  test 'saves searches when reporting on them' do
+    assert_difference "Search.where(saved: true).count", +1 do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.